CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 1.3.1
+
+* Removed the sorting function, as every terraformpy run would sort the duplicate keys
```

Comment:
Can we be more descriptive about the actual change so that readers can grok what changed in this version without actually reading the code?

Something like:

```suggestion
* When dumping JSON we no longer pass `sort_keys=True`, but rather rely on the stable insertion order of Python 3+ dictionaries
```